modules/ROOT/pages/push/binding.adoc

```diff
@@ -1,8 +1,36 @@
 include::{partialsdir}/attributes.adoc[]
 :service-name: Push Notifications
 
-= Binding a {mobile-client} with the {service-name} service
+== Prerequisites
+
+For *iOS*: you need an Apple Developer account and a provisioning certificate. Have a look at link:apple-setup.html[APNs Setup] for further instructions.
+
+For *Android*: A Google account and access to the Firebase console. See link:google-setup.html[Firebase Setup] to get started.
+
+== Binding a {mobile-client} with the {service-name} service
 
 include::{partialsdir}/generic-binding.adoc[]
 
+When you bind a mobile client to {service-name} a variant is created to store the credentials required to communicate with a push network.
```

Comment:
I think `{service-name}` shouldn't be here.

Reply:
i think it is, because when you bind it to a different service something different will happen (no variant will be created).